.github/workflows/ci.yml

```diff
@@ -72,12 +38,15 @@ jobs:
           apps: sbt
       - run: rm -r -fo src\sbt-test\skip-sbt1.4
       - run: rm -r -fo src\sbt-test\skip-windows
-      - run: sbt test-skip-windows scripted
+      - run: sbt ++${{ matrix.scala }} test-skip-windows scripted
         shell: bash
-  checks:
-    name: Scalafmt
+  formatting:
+    name: Scalafmt and Scalafix
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: coursier/setup-action@v1
+        with:
+          apps: sbt
       - run: ./bin/scalafmt --test
+      - run: sbt scalafixAll --check
```

Comment:
enforcing scalafix in this PR mostly for ExplicitResultTypes, to have deterministic types from the beginning for the sbt 2.x artifact